draft-shafranovich-rfc4180-bis.md

```diff
@@ -225,6 +225,10 @@ Example of a CSV file with nulls (if "NULL" is used to mark nulls):
    aaa,bbb,cccCRLF<br/>
    zzz,NULL,xxxCRLF
 
+## Empty files
+Implementers should be aware that in accordance to this specification a file
+does not need to contain any comment or record (empty file with zero bytes).
```

Comment:
Should this is plural?

"any comments or records"

Reply:
Probably yes. Done.